sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2IndexesResourceImpl.java

```diff
@@ -85,12 +84,18 @@ public Response addIndex(
       throw new WebApplicationException("columnName must be provided", Status.BAD_REQUEST);
     }
 
-    Schema.CqlTable table =
-        BridgeSchemaClient.create(blockingStub).findTable(keyspaceName, tableName);
-    if (table.getColumnsList().stream().noneMatch(c -> columnName.equals(c.getName()))) {
-      throw new WebApplicationException(
-          String.format("Column '%s' not found in table.", columnName), Status.NOT_FOUND);
-    }
+    bridge
+        .getTable(keyspaceName, tableName)
+        .flatMap(
+            table ->
+                table.getColumnsList().stream()
+                    .filter(c -> columnName.equals(c.getName()))
+                    .findAny())
+        .orElseThrow(
+            () ->
+                new WebApplicationException(
+                    String.format("Column '%s' not found in table.", columnName),
+                    Status.NOT_FOUND));
```

Comment:
This will be the error also if the table is not there.. do a `.map` instead of the `.flatMap` and move exception after the `.findAny()`